smdebug/tensorflow/base_hook.py

```diff
@@ -527,6 +527,8 @@ def _make_numpy_array(tensor_value):
 
     @staticmethod
     def _get_reduction_of_data(reduction_name, tensor_value, tensor_name, abs):
+        if hasattr(tensor_value, "numpy"):
+            tensor_value = tensor_value.numpy()
```

Comment:
This step, extracts the numpy value from EagerTensors

Reply:
how was it working before this change?

Reply:
Because of the incorrect `if abs:` check in the next fn call, we were always calling `np.absolute(numpy_data)`